app/index.html

```diff
@@ -6,7 +6,45 @@
         <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.css"/>
         <link rel="stylesheet" href="css/index.css"/>
     </head>
-    <body>
+
+    <body ng-app="webshopModule">
+        <nav class="navbar navbar-default">
+            <div class="container">
+                <div class="navbar-header">
+                    <a class="navbar-brand" href="#"><img src="http://lorempixel.com/120/40/abstract/" alt="Company Logo"></a>
+                </div>
+                <ul class="nav navbar-nav navbar-right">
+                    <li><a href="#">Sign Up</a></li>
+                    <li><a href="#">Log In</a></li>
+                </ul>
+            </div>
+        </nav>
+
+        <div class="container content">
+             <div><img src="http://placehold.it/1100x400/#cccccc" alt="Placeholder for Carousel"></div>
```

Comment:
Please make this a separate section of the page, and make the image it full-width.